### ISSUE TITLE
Random issues

### ISSUE THREAD

**Issue by jackpoz** [jackpoz/TrinityCore#80]
- Jinnai> I was trying to reproduce 2 things to create a issue but im unable to do so. I will try to ask here instead if thats okay - this might be something maybe Shauren can anwser : Is there any situation known that the core goes into negativ numbers when it should not?
Jinnai> I have two examples for this - number one is less important : https://cdn.discordapp.com/attachments/595592733196812289/602115820281266177/WoWScrnShot_072019_142916.jpg This is about damage - instead of nearly zero or zero, player got hit for insane dmg.
Jinnai> first time was aquatic guardian from bfd (https://wotlk.evowow.com/?npc=6047)
Jinnai> second time was illusion of jandice barov (https://classic.wowhead.com/npc=11439/illusion-of-jandice-barov)
Jinnai> i tryed to do this with the guardians, creating 40 of them and let them hit me but nothing happend
- Jinnai> Second - and this one is more critical : A guy found a way with the auctioneer addon to mass add auctions. While mass inserting auctions, he rans out of money for deposit fees. As soon as this happens, he was provided with max capped gold and started so send it to random people.
Jinnai> I only have the chatlog short before ban and wasnt able to reproduce this myself : [Keo]: You see if you have auto click on Post Auction in Appraiser and you have dont have enough gold to post it it goes negative [Keo]: and what comes after 0 is Gold cap
Jinnai> database screenshot for the gold thing after he gave around half of it away to two other accouns : https://cdn.discordapp.com/attachments/411079131468529664/604790171187281934/unknown.png
- Riztazz> there was a way to dupe stuff because character wasnt saved properly
by abusing deleteion/login packets
deletion*
jackpoz> ah ok, is it still a thing ?
Riztazz> Not sure, told him to enable logging
basically player wasnt set to any session
https://pastebin.com/2FcBTpBs that was our fix for it, maybe it will give you a clue how to reproduce it because i couldnt do it back in the day
but i knew what they did by logging packets ;p
obviously different sources so i might be wrong

**Comment by TheWinchesters** [comment]
About the first one:
I know a case where a player can not receive, but hit an abnormal of damage.
If it useful for this case, I can investigate how.

**Comment by jackpoz** [comment]
about the first one could you try after https://github.com/TrinityCore/TrinityCore/commit/a8b1b9a3c6266d589afb378f07c505f79c75a6a4 ? The test realm is up to date
